src/Symfony/Bridge/PhpUnit/Tests/DnsMockTest.php

```diff
@@ -18,15 +18,15 @@ class DnsMockTest extends TestCase
 {
     protected function tearDown()
     {
-        DnsMock::withMockedHosts(array());
+        DnsMock::withMockedHosts([]);
```

Comment:
PhpUnitBridge has to remain compatible with php 5.3. This change would break that compatibility.